sdk/react/src/markers/IconMarker/IconMarker.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+
+import * as React from 'react';
+import { Image, TouchableWithoutFeedback, View } from 'react-native';
+import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
+
+import styles from './IconMarker.styles';
+
+type Props = {
+  backgroundColor?: ?string,
+  iconUrl: ?string,
+  imageUrl: ?string,
+  onSeek: () => void,
+  style?: ViewStyleProp,
+  touchColor?: ?string,
+};
+
+type State = {
+  isExpanded: boolean,
+};
+
+export default class IconMarker extends React.Component<Props, State> {
+  static defaultProps = {
+    backgroundColor: undefined,
```

Comment:
Do you really need it? They are `undefined` by default

Reply:
Seems so, at least we get lint errors instead.